garrysmod/lua/includes/modules/undo.lua

```diff
@@ -355,7 +355,11 @@ end
 function Do_Undo( undo )
 
 	if ( !undo ) then return false end
+	
+	local call = hook.Run( "PreUndo", undo )

```

Comment:
Inline the if-statement instead of making this a local since it's only used in one place.